### DISCUSSION TITLE
Allow users to select a folder when subscribing to a new feed

### DISCUSSION THREAD

**Discussion by deanedridge** [ViennaRSS/vienna-rss#1163] [Ideas]
When one subscribes to a new feed using Vienna, there does not seem to be a way to choose which folder the feed will go into. The new feed seems to go into whichever folder was used last. It would be good to be able to change this via the dialog while subscribing as often the user does not wish to add the new feed to the last-used folder.

**Comment by DavidMunch** [comment]
To add to this; It would also be nice with some kind of small animation as to where the feed goes. Currently it just kind of disappears into my sea of feeds, where I then have to go look for it as descriped above.

Such as, when the user click 'Add feed', the text jumps out of the box and flies down into list, where the new entry appears.
Or, the new entry in the list oscilates in blue background color to indicate where it is. (Probably better)
Or something gimmicky/flashy like that.

**Comment by DavidMunch** [comment]
Bumping to remove the stale designation. This is a bad UX that needs fixing.
